package.json

```diff
@@ -25,16 +28,21 @@
     "@graphql-codegen/typescript": "^4.0.9",
     "@graphql-codegen/typescript-operations": "^4.2.3",
     "@graphql-codegen/typescript-react-apollo": "^4.3.0",
+    "@lingui/format-json": "^4.14.0",
```

Comment:
do you still need these lingui packages?

Reply:
Oh good catch Brian. Thanks so much. I thought I removed them already. I'll remove them.